vendor/ks-technologies/kst5420.yaml

```diff
@@ -0,0 +1,71 @@
+name: Impulse
```

Comment:
If possible, could you provide a description of the product in the name too? So something along the lines of `Impulse - Coverage Tool`

Reply:
No problem, just updated the name with a description in the latest commit @LDannijs, do I need to issue another PR?

Reply:
No you do not need to. I'll just review it again.